privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -654,7 +654,7 @@ def testconnection(cls, param):
         
         return success, desc
 
-    def add_user(self, uid, object_class=None, attributes=None):
+    def add_user(self, uid, object_class=None, attributes=None, dn=None):
```

Comment:
HI Martin,

maybe you should open up an issue, so that we can take the discussion there. I think it is a bit misplaced in this merged PR ;-)

You can map anything to anything else. Just like you wish.

There is already an example about expired users:
http://privacyidea.readthedocs.org/en/latest/configuration/useridresolvers.html?highlight=expired#expired-users

setting such a mapping simply introduces a new attribute in the privacyIDEA user object.

I.e. if you set the mapping

    "my_PI_is_locked": "ldap_locked_attribute"

then the user listing "GET /users" will contain an attribute "my_PI_is_locked" filled with the content of "ldap_locked_attribute" from the LDAP.

So the question is simply, what you will do with this new user attribute.
(Just try it out: Set a new attribute mapping in you resolver and fetch the users while you use your debug mode in the browser)

Adding Users: When adding a user you already can choose in which resolver to add the user.

Kind regards
Cornelius